pkg/schema/type.go

```diff
@@ -105,10 +104,7 @@ func (n NullType) GetDefaultValue() interface{} {
 }
 
 func (a AnyType) GetDefaultValue() interface{} {
-	if a.ValueType == nil {
-		return nil
-	}
-	return a.ValueType.GetDefaultValue() // delegate GetDefaultValue() functions will make defensive copy 👍
+	return a.defaultValue
```

Comment:
I think the onus is back on this function to provide a defensive copy.

Reply:
Deep copy for when interface is a Node, but scalar values are copied through interface type.go:110